src/sentry/models/organization.py

```diff
@@ -82,6 +82,7 @@ class Organization(Model):
     flags = BitField(flags=(
         ('allow_joinleave', 'Allow members to join and leave teams without requiring approval.'),
         ('enhanced_privacy', 'Enable enhanced privacy controls to limit personally identifiable information (PII) as well as source code in things like notifications.'),
+        ('disable_shared_issues', 'Enable sharing of limited details on issues to anonymous users.'),
```

Comment:
To clarify, the flag is "disable", and the help text says "enable". It's backwards.